knative/client.go

```diff
@@ -1,43 +1,68 @@
 package knative
 
 import (
-	"bytes"
 	"fmt"
-	"io"
-	"os"
 	"time"
 
 	"k8s.io/client-go/tools/clientcmd"
-	"knative.dev/client/pkg/kn/commands"
+	clienteventingv1beta1 "knative.dev/client/pkg/eventing/v1beta1"
 	clientservingv1 "knative.dev/client/pkg/serving/v1"
+	eventingv1beta1 "knative.dev/eventing/pkg/client/clientset/versioned/typed/eventing/v1beta1"
+	servingv1 "knative.dev/serving/pkg/client/clientset/versioned/typed/serving/v1"
 )
 
 const (
 	DefaultWaitingTimeout = 60 * time.Second
 )
 
-func NewClient(namespace string, verbose bool) (clientservingv1.KnServingClient, io.Writer, error) {
+func NewServingClient(namespace string) (clientservingv1.KnServingClient, error) {
```

Comment:
Looks good ! As a future step we might consider to take over those constructors methods to the knative client, too, so that the user doesn't have to specify the config on her own.